{{cookiecutter.project_slug}}/requirements/requirements.in

```diff
@@ -1,3 +1,6 @@
+# Maintain exact constraints from the base image.
+-r /opt/base-requirements.txt
+
 # Flask and related plugins
 flask
 flask-cors
```

Comment:
:question: :wink:

Reply:
Hmm? :) We use [Flask-Cors](https://pypi.org/project/Flask-Cors/) in all the services?

Reply:
But again, they are already in `base-requirements.in` :)